index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/headline.css">
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="left-section">
+        <p class="logo">My Logo</p>
+      </div>
+      <button class="right-section">
+        <img class="icon-menu" src="/Normal Button/Tertiary/Icons/Icon - Menu.png" alt="">
```

Comment:
- Kindly make sure you use the appropriate path for the hamburger  menu image as it is shown in the picture below your image is broken please fix that and make sure your image is displayed as expected it will make your page look nice :+1: 

![image](https://user-images.githubusercontent.com/95297251/205048772-bf468db1-e582-4425-84ad-60a2868f76a5.png)

Reply:
I have used appropriate path for the logo image. Sorry for my mistake.
Thank you